bot/cogs/watchchannels/bigbrother.py

```diff
@@ -42,7 +42,7 @@ async def watched_command(self, ctx: Context, update_cache: bool = True) -> None
         The optional kwarg `update_cache` can be used to update the user
         cache using the API before listing the users.
         """
-        await self.list_watched_users(ctx, update_cache)
+        await self.list_watched_users(ctx, update_cache=update_cache)
```

Comment:
Since it is implemented in the parent class, maybe also allow ordering by oldest first in Big Brother as well?

Reply:
I don't see why not, sure